dynamiqs/utils/operators.py

```diff
@@ -513,3 +513,32 @@ def hadamard(n: int = 1) -> Array:
     H1 = jnp.array([[1.0, 1.0], [1.0, -1.0]], dtype=cdtype()) / jnp.sqrt(2)
     Hs = jnp.broadcast_to(H1, (n, 2, 2))  # (n, 2, 2)
     return tensor(*Hs)
+
+
+def cnot() -> Array:
+    r"""Returns the CNOT operator.
+
+    It is defined by $\CNOT = \begin{pmatrix} 1 & 0 & 0 & 0 \\
+    0 & 1 & 0 & 0 \\
+    0 & 0 & 0 & 1 \\
+    0 & 0 & 1 & 0 \end{pmatrix}$.
+
+    Returns:
+        (array of shape (4, 4)) CNOT operator.
```

Comment:
The `_` ensures the typing is italicized in the documentation, it's part of our quirks 😉
```suggestion
        _(array of shape (4, 4))_ CNOT gate.
```